frontend/src/i18n/messages/en/index.ts

```diff
@@ -280,15 +280,15 @@ export const messages = {
         "If you’re interested in contributing to the open-source project or exploring the details of exactly what we’re building, check out the project at <strong><github>https://github.com/HHS/simpler-grants-gov</github></strong> or join our community at <strong><wiki>wiki.simpler.hhs.gov</wiki></strong>.",
     },
   },
-  Newsletter: {
+  Subscribe: {
     page_title: "Newsletter | Simpler.Grants.gov",
-    title: "Newsletter signup",
+    title: "Subscribe to project updates",
     intro: "Subscribe to get Simpler.Grants.gov project updates in your inbox!",
     paragraph_1:
       "If you sign up for the Simpler.Grants.gov newsletter, we’ll keep you informed of our progress and you’ll know about every opportunity to get involved.",
```

Comment:
Could you change this to "If you sign up for the Simpler.Grants.gov project updates, we'll keep you..." ? Replacing the word "newsletter."

Reply:
good catch - here's the update:
![image](https://github.com/navapbc/simpler-grants-gov/assets/93001277/2ed24614-f46a-4fe8-a07d-dbfd176ab5ef)

Reply:
Note that the title of pages still shows Newsletter 
![image](https://github.com/navapbc/simpler-grants-gov/assets/93001277/8c57c567-4bc6-46b1-9bcd-1787c4e24c87)

Reply:
Oh, we'll want to change that! I realize this is different than `subscribe` and @andycochran and @crystabelrangel may have different thoughts on this, but could we change that to say "Mailing List"? We can always change it something else if Design thinks it's better as a different name.

Reply:
"Project Updates"? Not sure if that is too redundant.